README.md

```diff
@@ -1,11 +1,38 @@
 ### appliance-os
 
-Ansible playbooks to configure a Kubernetes cluster.
+Ansible playbooks to configure a Kubernetes cluster node/master.
 
-Edit the [hosts.yml](hosts.yml) file with the correct IPs and users.
 
-Run the playbook:
+Start Multipass machine locally:
+
+```bash
+multipass launch -d 16G -m 4G -n os-builder -c 4
+```
+
+Paste your public key in:
 
+```bash
+multipass exec os-builder nano .ssh/authorized_keys
 ```
-ansible-playbook -i hosts.yml  playbook.yml
+
+Retrieve the machine IP
+
+```bash
+multipass info os-builder | grep IPv4
 ```
+
+Edit the [hosts.ini](inventory/sample/hosts.ini) file with the retrieved IP.
```

Comment:
Edit the [hosts.yml](inventory/sample/hosts.yml) file and set this IP as the `ansible_host` value for all hosts.

Reply:
What do you think about setting the IP in the vars file and we can refer to it in the **hosts.yml** file?

Reply:
Sure. let's do this.